internal/vmpool/pool.go

```diff
@@ -2,13 +2,19 @@ package vmpool
 
 import (
 	"context"
+	"sync"
 )
 
+const RunnerName = "drone-runner-aws"
```

Comment:
the runner should not be a constant, it is the name of the machine that starts the runner https://github.com/drone-runners/drone-runner-aws/blob/4a3a405b58f4c1293275d2ecc8b41f926e67cba0/command/delegate/delegate.go#L60

Reply:
This is not a runner name. This is actually used for the "creator" tag. It must be fixed.